.travis.yml

```diff
@@ -5,18 +5,26 @@ node_js:
   - "4"
   - "6"
   - "8"
+  - "9"
+
+git:
+  depth: false
+
+env:
+  - STAN_VERSION=v0.8.0-beta
```

Comment:
You could also use v0.7.2 to use the latest stable version.

Reply:
Excellent catch on the return - tests seem to run correctly with the beta. More important is the triggering of node-nats-streaming on node-nats changes.